+srv/expServer.m

```diff
@@ -14,17 +14,18 @@ function expServer(useTimelineOverride, bgColour)
 %       harware file is used.
 %
 %   Key bindings:
-%     t - Toggle Timeline on and off.  The default state is defined in the
+%     't' - Toggle Timeline on and off.  The default state is defined in the
```

Comment:
You're not being consistent with the inverted commas.  I find this just adds clutter, especially as they're followed by a hyphen anyway.  But if you're going to use them everywhere please stick to one type of comma.